### PR TITLE
Added Docker for testing Postgres

### PR DESCRIPTION
**Pull request by mabentley85**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | Added docker-compose.yml file for Postgres

Similar to how database docker images are being added to the [yii-db](https://github.com/yiisoft/db) for testing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,7 +133,18 @@ The package is tested with [PHPUnit](https://phpunit.de/). For tests need Postgr
 - port: `5432`
 - name: `testdb`
 - user: `postgres`
-- password: `postgres` 
+- password: `postgres`
+
+#### Docker Image
+
+To easily set up a pre-configured PostgreSQL instance for testing you can use the [docker-compose.yml](https://docs.docker.com/compose/compose-file/) file that is in the `/tests/Support/docker/pgsql` folder.
+[PostgreSQL 15](/tests/Support/docker/pgsql/docker-compose.yml)
```
Comment (xepozz) [inline README.md:141]: Please don't store docker files inside subdirectories.
It's a package. Any configuration files in the root directory are alright.

Comment (mabentley85) [reply README.md]: @xepozz OK, I moved the docker-compose.yml to the root, also updated the Readme.

**Review by samdark — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    container_name: yii-db-migration-test-pgsql
+    image: postgres:15
+    restart: always
+    environment:
+      POSTGRES_DB: testdb
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - "5432:5432"
+    expose:
```
Comment (xepozz) [inline docker-compose.yml:15]: The port was already exposed on the line above

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Use postgres/example user/password credentials
```
Comment (xepozz) [inline docker-compose.yml:1]: What is it for?
The credentials are: 
* db: testdb
* user: postgres
* password: postgres

I'd remove it

**Review by xepozz — CHANGES_REQUESTED** [review]

**Review by xepozz — APPROVED** [review]

**Comment by xepozz** [comment]
> @xepozz Now that a docker-compose.yml is included in the [db-pgsql](https://github.com/yiisoft/db-pgsql) package would it be better to omit the docker-compose.yml that I have added in the pull request?
> 
> This would avoid duplicating something that already exists (since db-pgsql is dev dependency of this package), the docker container in the db-pgsql package could be used for testing, all that would need to be done is change the connection details [here](https://github.com/yiisoft/yii-db-migration/blob/ad11e93c1aa83dcafd50f53db2ad4e0b93f9adda/tests/src/Support/Helper/PostgreSqlHelper.php#L40-L44) to match those in the db-pgsql docker image.
> 
> Thoughts?

As you wish, but I think this PR is useful for developing this package.

**Pull request merged**